index.html

```diff
@@ -6,54 +6,76 @@
     <title>Home Page</title>
 <link rel="stylesheet" href="styles.css">
 </head>
-    <body class="main-container">
-        <div class="app-wrap">
+    <body>
+     <div class="app-wrap">
        <header class="app-header">
-           <h1 class="logo">Ifza Rasool</h1>
-           <a class="nav-bar"><img src="Icons/Union.png"></a>
-       </header>
+           <a class="logo">Ifza Rasool</a>
```

Comment:
- [ ] [OPTIONAL] Well done including a link on the logo text 👍. However if you notice most websites when you click the logo text it directs the users to the homepage. so you can specify a `href` back so it can have that functionality.

something like this would be nice. 
```suggestion
           <a href="/" class="logo">Ifza Rasool</a>
```